### ISSUE TITLE
Change link to visualize to refer the published version.

### ISSUE THREAD

**Issue by sam-e-ch** [zazuko/cube-creator#1254]
After visualize now also shows "DRAFT" tagged cubes if directly referenced, it is possible to have a link which works once published. To directly point to the version which is published, the need is to change the link to point to the version.

Initial Issue:

> The link to visualize.admin.ch which is shown after a publish job is completed doesn't work.
> Right now, the link is simply redirected to the landing page.
> 
> It seems visualize.admin.ch expects the version to be included in the link, but the generated link doesn't add it.

**Comment by l00mi** [comment]
The link is created as agreed on, it seems to be a regression on visualize, see mention on their side.

**Comment by l00mi** [comment]
@Rdataflow that is not up to us?

**Comment by l00mi** [comment]
@tpluscode how much effort is it to use the directly versioned link in cube-creator? I guess we don't have the version until it is finally uploaded, correct?

**Comment by tpluscode** [comment]
yes, @l00mi I think that's what we should do anyway. I makes sense that a specific publish job links to visualize of that same version and not latest

**Comment by l00mi** [comment]
Okay so we will fix it at both ends. Best to be robust!

**Comment by Rdataflow** [comment]
@ortnever this will fix your current issue...

**Comment by l00mi** [comment]
@tpluscode do we have an issue for that on the cube-creator side?

**Comment by l00mi** [comment]
Changes in the URL templates (routing) of visualize did break this again.
We will discuss about the adapted URL templating with IXT.

**Comment by Rdataflow** [comment]
@tpluscode this still seems to be an issue
broken link https://test.visualize.admin.ch/en/browse?dataSource=Int&type=dataset&dataset=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fwald-faostat-production-test002
observed in https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!bfeogd101gasimportexport-qr7pov4nxqv/publication/jobs/cube-project!!bfeogd101gasimportexport-qr7pov4nxqv!!jobs!!KRkyOdYSw7YkUSUYqiHJb

should be https://test.visualize.admin.ch/en/browse?dataSource=Int&type=dataset&dataset=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fwald-faostat-production-test002%2F9

**Comment by Rdataflow** [comment]
@dabo505 can you reopen this? THX

**Comment by tpluscode** [comment]
Actually, I click those links and they both open the list of datasets /cc @ptbrowne
